mlir/tools/rocmlir-gen/rocmlir-gen.cpp

```diff
@@ -2885,9 +2885,15 @@ static func::FuncOp createVerifierFunc(ModuleOp module, const KernelIF &kernel,
   func::FuncOp verifyFuncDecl;
 
   if (testElemType.isa<FloatType>()) {
-    auto thr_RMS = getF32Val(RMSThreshold.getValue());
+    constexpr float defaultRMSThreshold(0.00003f);
+    constexpr float defaultRMSThresholdFP16(0.001f);
+    float RMSThresholdValue =
+        testElemType.isF16() ? defaultRMSThresholdFP16 : defaultRMSThreshold;
+    if (RMSThreshold)
+      RMSThresholdValue = RMSThreshold.getValue();
+    auto thr_RMS = getF32Val(RMSThresholdValue);
```

Comment:
Can we not `auto` this?

Reply:
What do you mean `auto`?

Reply:
You're using `auto` in a context where it's easy and helps clarity to write the type out.

(The existing code used `Value` for one of these constants, and you even changed it to `auto`)